twisted/web/_http2.py

```diff
@@ -686,7 +686,18 @@ def openStreamWindow(self, streamID, increment):
         @type increment: L{int}
         """
         # TODO: Consider whether we want some kind of consolidating logic here.
-        self.conn.increment_flow_control_window(increment, stream_id=streamID)
+        try:
+            self.conn.increment_flow_control_window(
+                increment, stream_id=streamID
+            )
+        except h2.exceptions.StreamClosedError:
+            # The stream got reset and we haven't worked it out yet. The stream
+            # window doesn't matter now. We still want to increment the
+            # connection window though.
```

Comment:
dumb question ahead. why ? :)

we want or we should ?

is this behaviour discussed in the h2 rfc?